apps/prairielearn/src/lib/db-types.ts

```diff
@@ -506,7 +506,7 @@ export const AiGenerationPromptSchema = z.object({
   errors: z.array(z.string()),
   html: z.string().nullable(),
   id: z.string(),
-  prompt_type: z.enum(['initial', 'human_revision', 'auto_revision']),
+  prompt_type: z.enum(['initial', 'human_revision', 'auto_revision', 'manual_change']),
```

Comment:
maybe i should change "human_revision" to "human_prompted_revision" and "manual_change" to "direct_human_revision"? I don't exactly know what the exact terminology we should use is.